src/inboxsdk.d.ts

```diff
@@ -660,10 +661,9 @@ export interface ComposeView extends EventEmitter {
   getBccRecipients(): Array<Contact>;
   getBodyElement(): HTMLElement;
   getCcRecipients(): Array<Contact>;
-  getCurrentDraftID(): Promise<string | null>;
+  getCurrentDraftID(): Promise<string | null | undefined>;
   getElement(): HTMLElement;
-  getDraftID(): Promise<string | null>;
-  getMessageIDAsync(): Promise<string>;
```

Comment:
Note: I was unable to find a place where this method existed at runtime.

`message-view` and `gmail-message-view` have this method. `ComposeView` does not seem to have it.